pyhector/constants.py

```diff
@@ -136,7 +136,6 @@
     "HC_TAU": "tau",
     "HC_RHO": "rho",
     "HC_MOLARMASS": "molarMass",
-    "HC_CALCDATE": "calcDate",
```

Comment:
Removed in https://github.com/JGCRI/hector/commit/9be5ef8dc9f9411d708d522727e3ccdad88296a7